build.sbt

```diff
@@ -1,82 +1,38 @@
-javacOptions ++= Seq("-source", "1.8", "-target", "1.8", "-Xlint")
-
-lazy val commonSettings = Seq(
-  organization := "Dwolla",
-  homepage := Option(url("https://stash.dwolla.net/projects/OPS/repos/cloudflare-public-hostname-lambda")),
+lazy val buildSettings = Seq(
+  organization := "com.dwolla",
+  name := "cloudformation",
```

Comment:
The artifact name `cloudformation` ought to be something like `cloudflare-api-client`. (I think just calling it `cloudflare` would violate [their trademark terms of service](https://www.cloudflare.com/trademark/), but `cloudflare-api-client` uses the trademark in a descriptive way and would be fine? Idk, IANAL)